lingetic-nextjs-frontend/app/languages/[language]/NextButton.tsx

```
@@ -0,0 +1,23 @@
+  ref?: React.RefObject<HTMLButtonElement | null>;
+}
+
+export default function NextButton({
+  onNext,
+  isLastQuestion,
+  ref,
+}: NextButtonProps) {
+  return (
+    <button
+      ref={ref}
+      autoFocus
```

Biome (1.9.4)

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)